### DISCUSSION TITLE
How about a default action of "try aax and then aaxc"?

### DISCUSSION THREAD

**Discussion by philgoetz** [mkb79/audible-cli#70] [Ideas]
Some audiobooks can be downloaded only as aaxc, but I can't de-DRM aaxc files yet, so I'd rather download as aax when possible.
But it would be infeasible for me to make a list of which could and couldn't be downloaded as aax, and then issue commands to do so.
I've got 333 books in my library, roughly half free and half paid for.

I could just do "--all --aax", and then do "--all --aaxc", and then write a program to delete the duplicate aaxc files.
But it would save a lot of time if audible-cli could automatically try aax first, and then try aaxc if aax doesn't work.

**Comment by mkb79** [comment]
That’s a good idea. I'll think about how I can implement this! FYI: You can do `--all --aax --aaxc` in one command. 

Decode aaxc files are not  complicated. You need at least ffmpeg v4.4 (you find precompiled windows binarys with google ;))! I'wrote a proof of concept to do this with a plug-in script for audible-cli. You can find it [here](https://github.com/mkb79/audible-cli/blob/master/plugin_cmds/cmd_remove-encryption.py).

**Comment by mkb79** [comment]
I've downloaded my ffmpeg version from https://www.gyan.dev/ffmpeg/builds/
